doc/specs/vulkan/appendices/VK_AMD_shader_fragment_mask.txt

```diff
@@ -44,8 +44,8 @@ None.
 
 === New SPIR-V Capabilities
 
-  * <<spirvenv-capabilities-table-shaderfragmentmaskamd,
-    code:ShaderFragmentMaskAMD>>
+  * <<spirvenv-capabilities-table-fragmentmaskamd,
+    code:FragmentMaskAMD>>
```

Comment:
Passing this and the spirvenv.txt changes on to the AMD extension author for review. Otherwise this PR seems fine - thanks!